app/controllers/fines_controller.rb

```diff
@@ -5,7 +5,9 @@ class FinesController < ApplicationController
   before_action :authenticate_user!
 
   def index
-    @response = symphony_client.fines(current_user.patron_key)
-    @fines = @response['fields']['blockList'].map { |fine| Fine.new(fine) }
+    @fine_response = symphony_client.fines(current_user.patron_key)
+    @checkouts_response = symphony_client.checkouts(current_user.patron_key)
+    @fines = @fine_response['fields']['blockList'].map { |fine| Fine.new(fine) }
+    @checkouts = @checkouts_response['fields']['circRecordList'].map { |checkout| Checkout.new(checkout) }
```

Comment:
This is kind of a "simple green" approach to getting this going. We now have two blocking Symphony requests before returning a response to the user.

Reply:
👍 I'm good with it; I'd vote get it merged and then refactor.